packages/build_path_prefix_map/build_path_prefix_map.0.3/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+synopsis:
+  "An OCaml implementation of the BUILD_PATH_PREFIX_MAP specification"
+description: "https://reproducible-builds.org/specs/build-path-prefix-map/"
+maintainer: "Gabriel Scherer <gabriel.scherer@gmail.com>"
+authors: [ "Gabriel Scherer <gabriel.scherer@gmail.com>" ]
+license: "MIT"
+homepage: "https://gitlab.com/gasche/build_path_prefix_map"
+doc: "https://gitlab.com/gasche/build_path_prefix_map/blob/master/README.md"
+bug-reports: "https://gitlab.com/gasche/build_path_prefix_map/issues"
+dev-repo: "git+https://gitlab.com/gasche/build_path_prefix_map.git"
+depends: [
+  "dune" {>= "1.11"}
+  "crowbar" {with-test}
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
The opam file autogeneration feature of dune is bugged with (lang dune 2.6) and older.
```suggestion
  ["dune" "subst"] {dev}
```